Makefile

```diff
@@ -39,7 +39,7 @@ TEST_NAMESPACE ?= default
 TEKTON_VERSION ?= v0.30.0
 
 # E2E test flags
-TEST_E2E_FLAGS ?= --fail-fast -p --randomize-all --slow-spec-threshold=5m -timeout=1h -progress -trace -v
+TEST_E2E_FLAGS ?= -p --randomize-all --slow-spec-threshold=5m -timeout=1h -progress -trace -v
```

Comment:
@SaschaSchwarze0 is the removal of the `fail-fast` on purpose?

Reply:
Added that for debugging. Let me remove it.

Reply:
Done @qu1queee